App/app/src/main/java/at/tugraz/vaccinationpassport/MainActivity.kt

```diff
@@ -36,7 +58,27 @@ class MainActivity : AppCompatActivity() {
         toggle.syncState()
         supportActionBar?.setDisplayHomeAsUpEnabled(false)
 
+        val navigationView: NavigationView = findViewById(R.id.navigationView)
+        navigationView.setNavigationItemSelectedListener(this)
+        navigationView.bringToFront()
```

Comment:
@Glordir this line was missing. Because of it, the click was behaving strangely. Now also the click in the emulator works a lot better.